src/main/java/com/mraof/minestuck/client/model/armor/IronLassArmorModel.java

```diff
@@ -0,0 +1,31 @@
+package com.mraof.minestuck.client.model.armor;
+
+import com.mraof.minestuck.Minestuck;
+import com.mraof.minestuck.item.armor.IronLassArmorItem;
+import net.minecraft.resources.ResourceLocation;
+import software.bernie.geckolib.model.GeoModel;
+
+public class IronLassArmorModel extends GeoModel<IronLassArmorItem>
+{
+	/**
+	 * Model made by DORO with edits by Riotmode
+	 */
```

Comment:
This is not where the model is defined. Keep such credit comments in the json file(s) instead only.